### ISSUE TITLE
deleted

### ISSUE THREAD

**Issue by ghost** [AdguardTeam/AdGuardHome#1133]
deleted

**Comment by uservictor** [comment]
I think this function can give the users a chance to help, to improve AdGuardFilters without creating new accounts on Adguard forum or GitHub.

(sorry for my bad englisch)

**Comment by ArchiveBase** [comment]
https://issues.vuetifyjs.com
https://github.com/vuetifyjs/vuetify-issue-helper

**Comment by ameshkov** [comment]
Good idea, thank you!

I think here's what we should do:
1. Add a proper template for the AG SDN filter list: https://github.com/AdguardTeam/AdguardSDNSFilter
2. Show a simple submission form pre-filled with necessary information
3. Use Github API to search for similar issues and show them
4. Once the necessary information is supplied, open a Github "new issue" page and pass this information there (see how it's done on https://issues.vuetifyjs.com/)

**We're yet to define what information should be necessary to submit an issue**.

**Comment by ArchiveBase** [comment]
I thought to write this comment due below [1] **[2]** past months and yesterday i saw a user ask to block several pornographic websites in public telegram chat without consider every ages members on it, however i see adguard still doesn't offer proper way to report them.

> Can you please add an "Report" button like the "Block" button to the AdGuardHome Query Log, to make it easy and quick to report advertising Domains?
> after clicking "Report" it shoud redirect to a website with checkboxes for informations like:

> I think this function can give the users a chance to help, to improve AdGuardFilters without creating new accounts on Adguard forum or GitHub.

The report page should be easy recognizable for every adguard users and report procedure must be flexible with options. form contain page necessary to display below basic status using tickmarks or smth else about certain domain

- lf it blocked by AdGuard default DNS (advertising, tracking etc)/ or *AG SDN filter
- lf it blacklisted by browsing security web service [1]
- lf it blocked by parental control web service [2]

_Additionally, add an email field, it helps to track the status as well. you can use github integrated bot with helpdesk for AG SDN filter reports and directly create helpdesk tickets for [1][2] reports then you can update any comments to users if necessary; otherwise simply close the issue or ticket and set-up to send pre-configured email based on template to user_

And i expect milestone for this issue :)

**Comment by ameshkov** [comment]
> And i expect milestone for this issue :)

Well, assigned to v0.104 for now. It's not a high priority yet as I don't see many people trying to report anything.

**Comment by valaria247** [comment]
@ameshkov is there any way to check family dns blocking status without visit to adult site for now? above explanation about check, report page is good idea

**Comment by ameshkov** [comment]
> is there any way to check family dns blocking status without visit to adult site for now?

Hm, nope, there's none. Could you please create a feature request explaining what you'd like to have? I think we can extend the "Check the filtering" block with this info

**Comment by valaria247** [comment]
> > is there any way to check family dns blocking status without visit to adult site for now?
> 
> Hm, nope, there's none. Could you please create a feature request explaining what you'd like to have? I think we can extend the "Check the filtering" block with this info

Above @ghost has explained about extend the report page with such feature and it's good to add for "Check the filtering" item in adguardhome as you said. Both ideas good but "Check the filtering" limits to adguardhome users

Simply i mean about a box to enter the domain and click 'check' next display result whether it blocked or not. if not blocked then let users to report it

Should i create separate feature request for that while it explained above?

**Comment by ameshkov** [comment]
> Should i create separate feature request for that while it explained above?

Yeah, ideally it should be a different feature request if I am getting it right
